eslint/babel-eslint-tests/test/fixtures/eslint-plugin-import-eslint-8/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  parser: "@babel/eslint-parser",
+  parserOptions: {
+    requireConfigFile: false,
+    babelOptions: {
+      configFile: false,
+      cwd: __dirname,
```

Comment:
Added the `cwd` option so that the babel/eslint-parser will search preset-flow from the test fixtures, otherwise it will search from our project root, which apparently does not depend on `preset-flow`. 

Previously the preset-flow is resolved because yarn hoisted the `preset-flow` required by the `benchmark` packages to the top level. The yarn hoisting behaviour is likely affected by the node.js 23 path changes, which just happens to reveal unsound imports that would have only been caught by the Yarn PnP.